requirements.txt

```diff
@@ -4,7 +4,9 @@
 broker==0.1.35
 cryptography==36.0.2
 deepdiff==5.7.0
-dynaconf==3.1.7
+# dynaconf==3.1.7 <- The latest dynaconf for vault changes is not released, using git based
+git+https://github.com/rochacbruno/dynaconf.git@master#egg=dynaconf
```

Comment:
Have you requested a release with those updates? I'm hesitant to go with unreleased master branch as our dependency.

Reply:
I'd say your fork is a better option than master, neither are as good as a release.

These requirements alone don't seem to be giving me enough to run `vault_login` script, as the executable for vault isn't included. 

Are you expecting users to have added the hashicorp repo and installed via DNF? 
https://learn.hashicorp.com/tutorials/vault/getting-started-install

Reply:
Yes! I am expecting people to install vault themselves, depending on their OS. The installation differs for different OSes.

Reply:
3.1.8 is out (and there's a pyup bot PR available with it) so this change can be dropped.

Reply:
@jyejare thanks for pushing for the dynaconf release!